.github/workflows/lint.yaml

```diff
@@ -16,22 +16,18 @@ jobs:
       - uses: "bewuethr/yamllint-action@v1.2.2"
         with:
           config-file: ".yamllint"
-      - uses: "pnpm/action-setup@v3"
+      - uses: "pnpm/action-setup@v4"
```

Comment:
Simplifying `pnpm` installation according to current instructions from the pnpm docs